api/src/main/java/org/openmrs/module/metrics/api/utils/EventsUtils.java

```diff
@@ -57,4 +70,41 @@ public static String readResourceFile(String file) throws MetricsException {
 			throw new MetricsException(e);
 		}
 	}
+	
+	public static MetricEvent buildEventObject(String simpleName, String uuid, Event.Action eventAction,
+	        EventConfiguration eventConfiguration) {
+		if (!eventConfiguration.isEnabled()) {
+			LOGGER.debug("Skipped writing '{}' Event to the database because "
+			        + "the synchronization for this object is disabled in the configuration", simpleName);
+			return null;
+		}
+		
+		StringBuilder tags = new StringBuilder(eventAction.name());
```

Comment:
Do we need a StringBuilder if this is only going to be a single value?

Reply:
yes true